go.mod

```diff
@@ -7,7 +7,6 @@ require (
 	github.com/boltdb/bolt v1.3.1 // indirect
 	github.com/hashicorp/go-hclog v0.9.1
 	github.com/hashicorp/go-msgpack v0.5.5
-	github.com/hashicorp/raft-boltdb v0.0.0-20171010151810-6e5ba93211ea
+	github.com/hashicorp/raft-boltdb v0.0.0-20210326213752-22de16c8e211
```

Comment:
It looks like `hashicorp/raft-boltdb` and `boltdb/bolt` are only used for `fuzzy` tests. The dependency is a bit troublesome too, because it's a circular dependency (raft depends, raft-boltdb which in turns depends on raft!).

I'd suggest breaking out `fuzzy` as its own go module, and have it depend on raft and raft-boltdb.  Doing so avoids requiring raft consumers from depending on boltdb unnecessarily.

Reply:
Thank you for this suggestion and I'll get that taken care of <3